packages/cascara/src/structures/AdminStructure/AdminStructureHeader.js

```diff
@@ -9,7 +9,7 @@ const TestLogo =
 
 const propTypes = {
   logo: pt.node,
-  title: pt.oneOfType([pt.arrayOf(pt.node), pt.node]).isRequired,
+  title: pt.oneOfType([pt.arrayOf(pt.node), pt.node]),
```

Comment:
```suggestion
  title: pt.oneOfType([pt.arrayOf(pt.node), pt.node]).isRequired,
```
This will definitely be required when using this Header slot component.